annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/WebAnnotationSearchRest.java

```diff
@@ -82,6 +82,10 @@
 		throw new ParamValidationException(ParamValidationException.MESSAGE_BLANK_PARAMETER_VALUE,
 			I18nConstants.ANNOTATION_VALIDATION,
 			new String[] { WebAnnotationFields.PARAM_QUERY, queryString });
+	    if (!SearchProfiles.contains(profile))
```

Comment:
this is not a proper implementation, the profile can be sent though the prefer header, see also the getProfile() method used bellow.